perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -269,12 +269,21 @@
 								ac.getResourceById(parms.readInt("resource")),
 								ac.getMemberById(parms.readInt("member")));
 					}
-				}  else if (parms.contains("group")) {
+				} else if (parms.contains("group")) {
 					if (parms.contains("workWithGroupAttributes")) {
-						return ac.getAttributesManager().getAttributes(ac.getSession(),
-								ac.getResourceById(parms.readInt("resource")),
-								ac.getGroupById(parms.readInt("group")),
-								parms.readBoolean("workWithGroupAttributes"));
+						if (parms.contains("attrNames")) {
+							return ac.getAttributesManager().getAttributes(ac.getSession(),
```

Comment:
When you add new combination of parameters for RPC method, you must add it to the special kind of javadoc above. See above ``getAttributes`` enum, find existing combination for group_resource attributes and add new entry below with added param ``attrNames``. Thank you.